### PR TITLE
BIG-15711: Add delete all functionality for products, categories, & bran...

### PR DESCRIPTION
**Pull request by npadilla**
...ds

cc: @aleachjr @lord2800

### CODE REVIEW

File: src/Bigcommerce/Api/Client.php [hunk 1]
```diff
@@ -491,6 +491,16 @@ public static function deleteProduct($id)
     }
 
     /**
+     * Delete all products.
+     *
+     * @return hash|bool|mixed
+     */
+    public static function deleteProducts()
```
Comment (lord2800) [inline src/Bigcommerce/Api/Client.php]: Perhaps `deleteAll(Products|Categories|Brands)`? Just so it's explicit that what you're deleting is everything. May not be important though.

**Comment by lord2800** [comment]
Otherwise, seems good to me.

File: src/Bigcommerce/Api/Client.php (hunk 1 above)
Comment (npadilla) [inline src/Bigcommerce/Api/Client.php]: good idea, will update :) thanks!

**Comment by npadilla** [comment]
done, see commit https://github.com/bigcommerce/bigcommerce-api-php/commit/76fd295fa53f91fd2719acbf57c09b205c3dd189. thanks you guys! :smiley:

**Comment by aleachjr** [comment]
:+1: pending Travis

**Pull request merged**